test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -2584,6 +2615,29 @@ public static Settings remoteStoreClusterSettings(
         return settingsBuilder.build();
     }
 
+    public static Settings remoteStoreClusterSettings(
+        String segmentRepoName,
+        Path segmentRepoPath,
+        String translogRepoName,
+        Path translogRepoPath,
+        String clusterRepoName,
+        Path clusterRepoPath
```

Comment:
Is this clusterStateRepo or routingTableRepo?

Reply:
It is routingTableRepo, refactored it.